census/requirements.txt

```diff
@@ -1 +1 @@
-tensorflow==1.2.0
+tensorflow==1.4.0rc0
```

Comment:
We should now be using `tensorflow==1.4.0`.

Reply:
Ok made changes.